### PR TITLE
Upgrade fix

### PR DESCRIPTION
**Pull request by NEETweeb**
fixed instructions in manually upgrading ce to ee

### CODE REVIEW

File: pro/pro-setup.rst [hunk 1]
```diff
@@ -67,26 +67,37 @@ Upgrade to Pro from Community Edition
...
+Also change ``SERVER_IMAGE_TAG`` in netmaker.env to ``<version>-ee``. For example: ``SERVER_IMAGE_TAG=v0.21.2-ee`` 
 
 Also change the ``INSTALL_TYPE`` from ce to pro.
 
 Then you will need to get the docker-compose pro file from here
 
 .. code-block::
 
-    wget -O docker-compose.override.yml https://github.com/gravitl/netmaker/blob/master/compose/docker-compose.pro.yml
+    wget -O docker-compose.override.yml https://raw.githubusercontent.com/gravitl/netmaker/master/compose/docker-compose.pro.yml
```
Comment (abhishek9686) [inline pro/pro-setup.rst:92]: wget -o /root/docker-compose.override.yml

File: pro/pro-setup.rst [hunk 2]
```diff
@@ -67,26 +67,37 @@ Upgrade to Pro from Community Edition
...
+    NETMAKER_TENANT_ID="<tenant id>"
+
+Make sure that it should look something like this:
+
+.. code-block::
 
-    LICENSE_KEY: “<license key>”
-    NETMAKER_TENANT_ID: "<tenant id>"
+    LICENSE_KEY=mylicensekeymylicensekeymylicensekeymylicensekeymylicensekey
+    NETMAKER_TENANT_ID=myguidte-nant-idmy-guid-tenantidmygu
```
Comment (abhishek9686) [inline pro/pro-setup.rst:82]: why random text, is this required ?

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: pro/pro-setup.rst (hunk 2 above)
Comment (NEETweeb) [reply pro/pro-setup.rst:82]: no. I just thought that some might include the quotes ("). Should I remove them?

Comment (abhishek9686) [reply pro/pro-setup.rst:82]: LICENSE_KEY: <license key>
NETMAKER_TENANT_ID: <tenant id>

Comment (NEETweeb) [reply pro/pro-setup.rst:82]: but this is an env file :)

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**